master/buildbot/test/unit/test_worker_ec2.py

```diff
@@ -22,8 +24,6 @@
     boto = None
     ec2 = None
 
-from twisted.trial import unittest
```

Comment:
pylint suggested it:

`C: 26, 0: external import "from twisted.trial import unittest" comes before "import boto" (wrong-import-order)`